Porting/perlgov-team-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env perl
+use v5.30.0;
+use warnings;
+use JSON::PP;
+
+my $file = 'Porting/core-team.json';
+my $data = JSON::PP->new->decode(scalar `cat $file`);
+
+my $pod = q{};
+
+for my $key (qw( active inactive )) {
+  $pod .= qq{=head2 \u$key Members\n\n=over 4\n\n};
+
+  my @items = map {; "<$_>" } $data->{$key}->@*;
+
+  open(my $fh, '-|', 'git', 'check-mailmap', @items)
+    or die "error running check-mailmap: $!";
+
+  my @lines = <$fh>;
+
+  $pod .= "=item $_\n" for sort @lines;
```

Comment:
Recommend a `use locale` for this sort. Though it also may require decoding from UTF-8 before sort + encoding the final output to UTF-8 for that to work properly.

Reply:
This seems like a bad idea.  It would mean that the sort might do quite a bit of re-ordering if different runners of the script had different locales, no?  Simpler, I think, to just leave things in a single, stable order, even if it's not what any single phone book editor would pick.